tools/js-sdk-release-tools/src/xlc/codeUpdate/updateUserAgent.ts

```diff
@@ -0,0 +1,48 @@
+import { ApiVersionType, SDKType } from "../../common/types";
+import { getSDKType } from "../../common/utils";
+import { logger } from "../../utils/logger";
+
+const fs = require('fs');
+const path = require('path');
+
+export function changeClientFile(packageFolderPath: string, packageVersion: string){
+    const packageJsonData: any = JSON.parse(fs.readFileSync(path.join(packageFolderPath, 'package.json'), 'utf8'));
+    const packageName = packageJsonData.name.replace("@azure/", "");
+    const sdkType = getSDKType(packageFolderPath);
+    let files: string[];
+    switch (sdkType) {
+        case SDKType.HighLevelClient:
+            // Update version in src for HLC
+            files= fs.readdirSync(path.join(packageFolderPath, 'src'));
```

Comment:
since you already know the file name of client, you should limit the update in that file.

1. check the existence of the file https://github.com/Azure/azure-sdk-tools/blob/d5ac6007eef9aaeb9431e56d26e65161a01b579a/tools/js-sdk-release-tools/src/mlc/apiVersion/apiVersionTypeExtractor.ts#L19
2. search and update the user agent pattern